src/core/vscore.cpp

```diff
@@ -1799,6 +1799,16 @@ VSCore::VSCore(int flags) :
     cpuLevel(INT_MAX),
     memory(new MemoryUse()),
     enableGraphInspection(flags & ccfEnableGraphInspection) {
+#ifdef VS_USE_MIMALLOC
+    static std::once_flag mionce;
+    std::call_once(mionce, []()
+    {
+        mi_option_set_default(mi_option_segment_cache, 32);
+        mi_option_set_enabled_default(mi_option_large_os_pages, 1);
+        mi_option_set_enabled_default(mi_option_eager_commit, 0);
```

Comment:
Why disable eager commit?

Reply:
In theory, it moves the page fault to the threadpool, preventing (a) serial bottlenecks, and (b) NUMA imbalances, but... Windows doesn't support overcommit, so it doesn't matter.